### PR TITLE
Prompt neutrons 2.0

### PR DESCRIPTION
**Pull request by cfichtlscherer**
This is a follow-up on the pull request [prompt neutrons](https://github.com/openmc-dev/openmc/pull/2327).

### CODE REVIEW

File: openmc/settings.py [hunk 1]
```diff
@@ -1722,6 +1748,7 @@ def from_xml_element(cls, elem, meshes=None):
         settings._ufs_mesh_from_xml_element(elem, meshes)
         settings._resonance_scattering_from_xml_element(elem)
         settings._create_fission_neutrons_from_xml_element(elem)
+        settings._create_delayed_neutrons_from_xml_element(root)
```
Comment (paulromano) [inline openmc/settings.py:1751]:
```suggestion
        settings._create_delayed_neutrons_from_xml_element(elem)
```

File: openmc/settings.py [hunk 2]
```diff
@@ -224,6 +224,10 @@ class Settings:
     weight_windows : WeightWindows iterable of WeightWindows
         Weight windows to use for variance reduction
 
+        .. versionadded:: 0.13.3
+    create_delayed_neutrons : bool
+        Whether delayed neutrons are created in fission.
+
         .. versionadded:: 0.13
```
Comment (paulromano) [inline openmc/settings.py:231]:
```suggestion
        .. versionadded:: 0.13
    create_delayed_neutrons : bool
        Whether delayed neutrons are created in fission.

        .. versionadded:: 0.13.3
```

**Review by paulromano — CHANGES_REQUESTED** [review]

File: openmc/settings.py (hunk 1 above)
Comment (cfichtlscherer) [reply openmc/settings.py]: thanks!

**Review by paulromano — APPROVED** [review]
Looks good this time around. Thanks for the new feature @cfichtlscherer!

**Pull request merged**